en/flight_modes/offboard.md

```diff
@@ -75,13 +89,21 @@ See https://github.com/PX4/Firmware/pull/12149 and https://github.com/PX4/Firmwa
 
 ### Rover
 * [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED)
-  * The following input combinations are supported: <!-- https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
+  * The following input combinations are supported (in `type_mask`): <!-- https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
     * Position setpoint (only `x`, `y`, `z`)
+  * Specify the *type* of the setpoint in `type_mask` (not part of the MAVLink standard).
+    The values are:
+    - Following bits not set then normal behaviour.
+    - 12288: Loiter setpoint (vehicle stops when close enough to setpoint).

```

Comment:
@Jaeyoung-Lim What is "close enough"? Is there an acceptance radius defined somewhere?

Also see line above - what is "normal behaviour" - just tracking setpoint until actually on it according to GPS

Reply:
Yes, it will stop at the waypoint when it reaches the acceptance radius. Normal behavior would be that the setpoint is passed as a reference for the L1 controller

Reply:
1. Acceptance radius is [NAV_ACC_RAD](http://docs.px4.io/master/en/advanced_config/parameter_reference.html#NAV_ACC_RAD)?
2. I have no idea what this means:

> Normal behavior would be that the setpoint is passed as a reference for the L1 controller

Reply:
@Jaeyoung-Lim Can you confirm/answer?

Reply:
@hamishwillee Sorry for the late response hamish, the problem is that I would actually need to test all these setpoint types since I am not sure what they are doing